### PR TITLE
Add meetup guidance for COVID-19

### PR DESCRIPTION
**Pull request by mjang**
We should also link to this letter from somewhere, suggestion invited.

I feel like I had to keep this letter as general as possible, as conditions (and governmental recommendations) vary widely.

### CODE REVIEW

File: docs/organizer-guide/meetups/covid-19-meetups.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Meetups and COVID-19
+====================
+
+With the recent announcement about the Write The Docs NA (Portland) conference, we wanted to provide guidance for our community of meetups.
+
+We've received a number of questions on whether our meetups should continue to meet "in person" during the current situation with COVID-19.
```
Comment (meker12) [inline docs/organizer-guide/meetups/covid-19-meetups.rst]:
```suggestion
We've received a number of questions about whether Write the Docs community meetups should continue to host in-person events under the current circumstances related to the spread of COVID-19.
```

File: docs/organizer-guide/meetups/covid-19-meetups.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+For the meetup organizers among us, we urge you to follow the best information available from your local, state, provincial, and national governments, including health advisories. If they are discouraging in-person events, we urge you to follow their advice.
+
+If you and your groups wish to continue to meet, please consider using available remote tools in whole or part. That'll help us, as a community, to be more inclusive of those who are not able or willing to attend in person.
```
Comment (meker12) [inline docs/organizer-guide/meetups/covid-19-meetups.rst]:
```suggestion
If you and your groups wish to continue to meet, please consider using available remote tools in whole or part. That will help us, as a community, to be more inclusive of those who are not able or willing to attend in person.
```

File: docs/organizer-guide/meetups/covid-19-meetups.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+If you have any questions, please don’t hesitate to reach out in the `#meetup-organisers <https://writethedocs.slack.com/messages/meetup-organisers/>`_ Slack channel. We'll do our best to answer your questions.
```
Comment (meker12) [inline docs/organizer-guide/meetups/covid-19-meetups.rst]:
```suggestion
If you have any questions, don’t hesitate to message us in the `#meetup-organisers <https://writethedocs.slack.com/messages/meetup-organisers/>`_ Slack channel. We'll do our best to address your concerns.
```

**Review by meker12 — APPROVED** [review]
A few suggestions, but looks good overall.

**Comment by meker12** [comment]
> We should also link to this letter from somewhere, suggestion invited.
> 
> I feel like I had to keep this letter as general as possible, as conditions (and governmental recommendations) vary widely.

How about sending a message in the meetup organizers channel, and pinning the message to th channel.  Also, sending the text in a group email to meetup organizers with a link to the text on the writethedocs.org site where they can check for updates?

File: docs/organizer-guide/meetups/covid-19-meetups.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Meetups and COVID-19
+====================
+
+With the recent announcement about the Write The Docs NA (Portland) conference, we wanted to provide guidance for our community of meetups.
```
Comment (plaindocs) [inline docs/organizer-guide/meetups/covid-19-meetups.rst]: s/NA (Portland)/Portland/

**Review by plaindocs — COMMENTED** [review]
Should we include a short list of suggestions for remote meetups here? Email book club, remote speaker, collaborative writing....

**Comment by plaindocs** [comment]
What @meker said. We can tweet it out also, and link it in the meetup organizer guide.

**Review by shrielenee — APPROVED** [review]
I like @meker12 suggestions. Looks good!

File: docs/organizer-guide/meetups/covid-19-meetups.rst (hunk 4 above)
Comment (mjang) [reply docs/organizer-guide/meetups/covid-19-meetups.rst]: > Should we include a short list of suggestions for remote meetups here? Email book club, remote speaker, collaborative writing....

I'm tempted to push such questions to #wtd-meetup-remote

Comment (mjang) [reply docs/organizer-guide/meetups/covid-19-meetups.rst]: PS, I've warned one of the leaders of #wtd-meetup-remote, and await his concurrence.

Comment (mjang) [reply docs/organizer-guide/meetups/covid-19-meetups.rst]: I have concurrence to point to #wtd-meetup-remote for advice on howto remote meetup

**Review by plaindocs — APPROVED** [review]
👍

**Pull request merged**